src/contact/containers/EditInfo.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import EditInfo from '../components/EditInfo';
+import * as actions from '../actions/Actions';
+import { bindActionCreators } from 'redux';
+
+
+const mapStateToProps = (state) => {
+	return {
+		info : state.reducer.details,
+		hasError: true,
+		error: null,
```

Comment:
If you have to assign to true or null, you should be doing that inside the component, not in container binding.

Reply:
ok